app/main.py

```diff
@@ -42,10 +42,11 @@ def create_tables(engine, psql_environment):
 from app.routers import (  # noqa: E402
     about_us, agenda, calendar, categories, celebrity, credits,
     currency, dayview, email, event, export, four_o_four, friendview,
-    google_connect, invitation, login, logout, profile,
+    google_connect, invitation, joke, login, logout, profile,
     register, search, telegram, user, weekview, whatsapp,
 )
 
+load_jokes.load_daily_jokes(next(get_db()))
 json_data_loader.load_to_database(next(get_db()))
```

Comment:
Why not use json_data_loader() instead like the others?

Reply:
I have changed that.
Now I am using json_data_loader